src/components/register/index.test.js

```diff
@@ -31,4 +32,11 @@ describe('RegisterHOC', () => {
   it('should render Register', () => {
     expect(wrapper.find('Register')).to.have.lengthOf(1);
   });
+
+  it('should bind dialogDisplayed action to Register props.dialogDisplayed', () => {
```

Comment:
`setActiveDialog` is no longer used in register component, so it can be removed and this test too.